core/templates/dev/head/pages/collection_editor/editor_tab/CollectionLinearizerService.js

```diff
@@ -184,8 +184,11 @@ oppia.factory('CollectionLinearizerService', [
         var linearNodeList = _getCollectionNodesInPlayableOrder(collection);
         CollectionUpdateService.addCollectionNode(
           collection, explorationId, summaryBackendObject);
+        var skillName = summaryBackendObject.title;
```

Comment:
LGTM assuming we're using skills where we need to use skills, and skill IDs where we need to use skill IDs.